### ISSUE TITLE
Add support for model_field also a callable

### ISSUE THREAD

**Issue by nicam** [tfoxy/graphene-django-optimizer#51]
Hi There!

Would it be possible to have `model_field` also use `_normalize_hint_value` so we could use a lambda function?

We have the problem that we need to define the hints at runtime since they are different based on language. Using the annotation does only work with lambdas as they get executed during the request.

Thanks for your help!

**Comment by tfoxy** [comment]
Hi!

Let me know if you still need this to be possible. If so, can you provide some example on how would this be used?

**Comment by tfoxy** [comment]
From what I understand, you want something like the following:

```
    @gql_optimizer.resolver_hints(
        model_field=lambda info: 'product',
    )
    def resolve_product(root, info):
        return root.product
```

But instead of returning `product` in the lambda, you want to use `info`. Is that right?

**Comment by nicam** [comment]
actually just beeing able to use a lambda function, instead of a string, for model_field would already be enough :)

As far as I have seen you'd only need to wrap it with `_normalize_hint_value` internally

**Comment by nicam** [comment]
Concretly I'd like to do the following:

    @gql_optimizer.resolver_hints(
        model_field = lambda info:  f'image_{translation.get_language()}',
    )

The field name needs to be evaluated in the lambda since it differs by language so I can prefetch the correct model field

**Comment by tfoxy** [comment]
Understood! Yeah, from what I can see it should be a simple change. Let me check what I can do. Please ping me if I don't answer you by Sunday.

**Comment by tfoxy** [comment]
Fixed in v0.7.0!

**Comment by nicam** [comment]
Thanks a lot!
